from core.exceptions.CustomException import CustomException

class VlanException(CustomException):
   """ Raised when there is an issue with vlans """
   pass